src/busco/busco/test.sh

```diff
@@ -0,0 +1,23 @@
+test_dir="$meta_resources_dir/test_data"
+
+echo "> Running busco"
```

Comment:
Can you also add a test that runs the `--auto_lineage` parameter, to verify that the component still works in this configuration? :bow: